1.10/tutorials/dcos-101/app1.md

```diff
@@ -10,30 +10,30 @@ menu_order: 3
 <table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;"><b>Important:</b> Mesosphere does not support this tutorial, associated scripts, or commands, which are provided without warranty of any kind. The purpose of this tutorial is to demonstrate capabilities, and may not be suited for use in a production environment. Before using a similar solution in your environment, you must adapt, validate, and test.</td> </tr> </table>
 
 # Objective
-You now have a working persistence layer -redis- running in your cluster.
-In this section you deploy the first app connecting to redis. Note that this tutorial deliberately choose to focus on the principles and to deploy a very simple app with no further logic than connecting to redis.
+You now have a working persistence layer - [Redis](https://redislabs.com/) - running in your cluster.
+In this section you will deploy a simple app connecting to Redis. 
 
 # Steps
-* Check out app1
-  * Let us first take a short look at the [app](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.py). It is very simple and just checks whether it can reach redis and then prints the total number of keys stored in redis.
-* Deploy app 1
-  * The python script has several dependencies (python version and redis-python), which you cannot assume to be present on all agent nodes. Because of this, you should run it in the `mesosphere/dcos-101` Docker container that provides all of these dependencies. Feel free to take a look at the [DOCKERFILE](https://github.com/joerg84/dcos-101/blob/master/app1/DOCKERFILE), which was used to create the `mesosphere/dcos-101` image.
-  * Have a look at the [app definition](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.json). This app definition will download the python script and then run it inside the `mesosphere/dcos-101` Docker container.
-  * Add app to Marathon: `dcos marathon app add https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.json`
-* You have multiple options to check that app1 is running:
+* Review app
+  * Let us first take a short look at the [app](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.py). It is very simple and just checks whether it can reach Redis and then prints the total number of keys stored there.
+* Deploy app
+  * The python script has a dependency on the [redis-python](https://pypi.python.org/pypi/redis) library, which you cannot assume to be present on all agent nodes. Because of this, you should run it in the `mesosphere/dcos-101` Docker container that provides all of the dependencies. Feel free to take a look at the [DOCKERFILE](https://github.com/joerg84/dcos-101/blob/master/app1/DOCKERFILE), which was used to create the `mesosphere/dcos-101` image.
```

Comment:
Not sure why we're calling it "redis-python" in the link (is this a package name on mac or something?), elsewhere I've seen it referred to as redis-py or just the "Python client for Redis"